roles/postgresql/tasks/create_db.yml

```diff
@@ -20,7 +20,7 @@
     state: "present"
     login_user: "{{ postgres_admin_user }}"
     owner: "{{ item.owner | default(postgres_admin_user) }}"
-  delegate_to: '{{ postgres_host }}'
+  #  delegate_to: '{{ postgres_host }}'
```

Comment:
Is this no longer needed for creating databases from other roles?

Reply:
that is passed using the create_db module here. https://github.com/pulibrary/princeton_ansible/blob/5168670a3909a28377bce1dc76d773d32cbe75ba/roles/postgresql/tasks/create_db.yml#L6